### ISSUE TITLE
File to list export headers

### ISSUE THREAD

**Issue by EugeneZelenko** [include-what-you-use/include-what-you-use#292]
I think will be good idea to have command line option to specify file with list of export (compound) headers. In this case headers will not be peppered with IWYU specific pragmas.

**Comment by kimgr** [comment]
Yes, I agree. We've talked about extending the current mapping format, so there's a single configuration format for IWYU.
